final-project/README.md

```diff
@@ -5,6 +5,45 @@ This application allows users to sign up for an account, sign in, record and man
 
 The server and client are both written on Go, and run on Docker.
 
+## Project Directory
+```
+final-project/
+│   Taskfile.yml
+│   go.mod
+│   main.go                     - main file for running the server
+│   README.md
+│   routes.go
+│
+└───client/                     - module for client app
+│   
+└───deploy/
+│   │
+│   └───dev/
+│       │   
+│       └───data/
+│       │   │        
+│       │   └───data.json       - sample data for testing dev
+│       │ 
+│       │   .env                - environment variables for dev
+│       └───docker-compose.yml
+│   
+└───server/
+    │   auth/                   - package for user registration and authentication
+    │   categories/             - package for handling categories of transactions
+    │
+    └───data/
```

Comment:
Why not "storage"
I always try to avoid "data", it means everything and it's never explicit.